examples/tabs/skunkworks/NoDucksTabContainer.tsx

```diff
@@ -1,20 +1,20 @@
-import { get, initialResponseState, OfflineComponent } from "@misk/core"
-import * as React from "react"
-import { SampleTableComponent } from "../components"
+import { get, initialResponseState, OfflineComponent } from "@misk/core";
+import * as React from "react";
```

Comment:
These semi-colons everywhere look like Prettier didn't run. Try `miskweb lint -e` from the root directory to re-apply linting everywhere.

Reply:
Spoke offline with @adrw and realized the linting didn't work in `skunkworks` specifically. AI is to delete `skunkworks` in a future PR.